vars/pact.groovy

```diff
@@ -0,0 +1,20 @@
+import uk.gov.defra.ffc.Pact
+
+def string(credentialsId,variable){
+  Pact.string(this, credentialsId, variable)
+}
+
+def usernamePassword(credentialsId, usernameVariable, passwordVariable){
+  Pact.usernamePassword(this, credentialsId, usernameVariable, passwordVariable)
+}
+
+def pacts(repoName, string, usernamePassword){
```

Comment:
Maybe make this more descriptive, function names should be a verb.  eg publishPacts.